website/content/docs/ecs/get-started/requirements.mdx

```diff
@@ -7,16 +7,10 @@ description: >-
 
 # Requirements
 
-Currently, the following requirements must be met in order to install Consul on ECS:
+The following requirements must be met in order to install Consul on ECS:
 
 1. **Terraform:** The tasks that you want to add to the service mesh must first be modeled in Terraform.
-1. **Launch Type:** Only the Fargate launch type is currently supported.
+1. **Launch Type:** Fargate and EC2 launch types are supported.
 1. **Subnets:** ECS Tasks can run in private or public subnets. Tasks must have [network access](https://aws.amazon.com/premiumsupport/knowledge-center/ecs-pull-container-api-error-ecr/) to Amazon ECR to pull images.
-1. **Consul Servers:** Currently, Consul servers must run inside ECS on Fargate using the `dev-server` Terraform module. This is a development/testing only server that does not support persistent storage. In the future, we will support production-ready Consul servers running in HashiCorp Cloud Platform and on EC2 VMs.
-
-## Future Improvements
-
-- Support EC2 launch type.
-- Support production-ready Consul servers running outside of ECS in HashiCorp Cloud Platform or EC2.
-- Support Consul TLS, ACLs, and Gossip Encryption.
-- Support Consul service health checks.
+1. **Consul Servers:** You can use your own Consul servers or run servers inside ECS on Fargate using the `dev-server` Terraform module. The is a development/testing only server that does not support persistent storage.
```

Comment:
```suggestion
1. **Consul Servers:** You can use your own Consul servers (VM) or use HCP Consul. For development purposes or testing, you may use the the `dev-server` [Terraform module](https://github.com/hashicorp/terraform-aws-consul-ecs/tree/main). The is a development/testing only server that does not support persistent storage.
```
I think it's better if we make it clearer how Consul servers fit into this framework. Right now the only two non-dev options are Consul server running on VMs or HCP Consul.